.github/workflows/ci-deploy-env.yml

```diff
@@ -79,28 +80,19 @@ jobs:
           --tag $ECR_REGISTRY/$ECR_REPOSITORY:$BRANCH \
           --file ./Dockerfile ./
 
-#    - name: Kick off Terraform deploy in sysops/ if not live
-#      id: sysops-deploy
-#      run: |
-#        BRANCH=$(echo "${GITHUB_REF#refs/*/}" | sed 's/.*\///')
-#        if [[ "$BRANCH" != master ]]; then
-#          curl \
-#          -X POST \
-#          -H "Accept: application/vnd.github.v3+json" \
-#          https://api.github.com/repos/meedan/sysops/actions/workflows/deploy_${{ github.event.repository.name }}.yml/dispatches \
-#          -u ${{ secrets.SYSOPS_RW_GITHUB_TOKEN }} \
-#          -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}", "branch": "'$BRANCH'"}}'
-#        fi
-
-#   - name: Send Slack Notification
-#     id: slack-notify
-#     if: ${{ always() }}
-#     uses: 8398a7/action-slack@v3
-#     with:
-#       status: ${{ job.status }}
-#       author_name: 'Github Actions'
-#       icon_emoji: ':octocat:'
-#       fields: repo,message,author # selectable (default: repo,message)
-#     env:
-#       GITHUB_TOKEN: ${{ secrets.CONFIGURATOR_TOKEN }} # optional
-#       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }} # required
+  # trigger-sysops-deploy:
```

Comment:
@dmou Here we want to kick off the deployment if develop or master. In `inputs`, we want to pass different variables depending on the branch?

Reply:
what does this mean?

Reply:
I was looking in the ticket:

>  the API call will differ based on whether the workflow was started by merging to develop or to master, as these pass different variables to the API call that tell the workflow in sysops/ which environment to deploy

So I was wondering what exactly needed to be different here, and if we needed to pass different variables in this [line](https://github.com/meedan/pender/blob/846ce0e6df53a09399a1985e9b2ff6a99841a7ed/.github/workflows/ci-deploy-env.yml#L97):
```
  #         -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}", "branch": "'$BRANCH'"}}'
```

Reply:
I see, this is for line 97 and not 83! 

> Here we want to kick off the deployment if develop or master

yes, this is specified in lines 6-7. the workflow runs on pushes to branches that correspond to deployment environments.

> In inputs, we want to pass different variables depending on the branch?

the API call will only send the body we tell it to. if we don't tell it what pender branch it is originating from, it won't know what environment to deploy to! also, we try not to let service repos expose infrastructure details, so that's all handled in the sysops repo :slightly_smiling_face: